### PR TITLE
Issue 86 visualization to frontend

### LINKED ISSUE #51: File overview
An overview for all (user specific) uploads

- [x] #60
- [x] #61
- [x] #76
- [ ] Possible other improvements

### LINKED ISSUE #62: Transpiler
## Backend

From Feature Model .xml to g6 Graph Visualisation (https://g6.antv.vision/zh)

 - [x] #63
   - Impelemented XML to JSON and JSON to G6 (will be used in first ddueruemweb prototype)
   - However there is JSON to Graphology left. For graphology with dimension you need to process the model with the clojure script first
   - After this you can process it with the methods from transpiler
 - [x] #69
 - [x] #80

## Frontend
After implementation of transpiler in the choosen language, make it the view accessible in frontend
- [x] #86

### LINKED ISSUE #67: Create CRUD for Tags

### LINKED ISSUE #86: Add visualization to frontend

### LINKED ISSUE #89: Backend: Remove View all users
Remove 
```python
class UserViewSet(ModelViewSet):
```
from `backend/core/user/viewsets.py` because this is a huge data privcay problem

### PR DESCRIPTION
**Pull request by Eixix**
- Visualization on big screen
- minor usability additions (press enter to send form)
- static file serving in docker with nginx as a reverse proxy (SSL to be added)

To test do the following in the root directory:
```bash
git clone https://github.com/h3ssto/ddueruem-web/
cd ddueruem-web
git checkout issue-86-visualization-to-frontend
cp backend/ddueruemweb/.env.example backend/ddueruemweb/.env.production
docker-compose down -v
docker-compose up --build -d
docker exec -it ddueruem-web_backend_1 /api/manage.py createsuperuser             
```

Go to http://localhost:8080 and enjoy!
Note that before uploading a File,  there should be at least 1 (public) Tag  present

### CODE REVIEW

**Comment by h3ssto** [comment]
Missing:
* [x] Not logged in => Start page shows read-only list of all public feature models
* [x] Creating superuser yields `django.core.exceptions.ImproperlyConfigured: Set the SECRET_KEY environment variable`
* [x] Remove JSON from fronted popups

File: backend/core/fileupload/views.py [hunk 1]
```diff
@@ -83,3 +83,8 @@ def list(self, request, **kwargs):
 
     def perform_create(self, serializer):
         serializer.save(owner=self.request.user)
+
+    def delete(self, pk):
+      tag = self.get_object(pk)
+      tag.delete()
+      return Response(status=status.HTTP_204_NO_CONTENT)
```
Comment (h3ssto) [inline backend/core/fileupload/views.py:90]: Appropriate RC?

**Review by h3ssto — DISMISSED** [review]

File: backend/core/fileupload/views.py (hunk 1 above)
Comment (wurstbroteater) [reply backend/core/fileupload/views.py:90]: yes

**Review by wurstbroteater — APPROVED** [review]
looki goodi

**Pull request merged**